src/java/org/apache/cassandra/db/commitlog/AbstractCommitLogSegmentManager.java

```diff
@@ -506,7 +506,8 @@ public void awaitTermination() throws InterruptedException
         for (CommitLogSegment segment : activeSegments)
             segment.close();
 
-        bufferPool.emptyBufferPool();
+        if (bufferPool != null)
+            bufferPool.emptyBufferPool();
```

Comment:
I have not checked if `org.apache.cassandra.streaming.LongStreamingTest` also fail in 3.11 but we should probably backport that fix anyway.

Reply:
Doesn't fail there, but I can indeed add it.